nservicebus/rabbitmq/delayed-delivery.md

```diff
@@ -6,31 +6,31 @@ reviewed: 2017-03-02
 versions: '[4,]'
 ---
 
-In Versions 4.3 and above, the RabbitMQ transport no longer relies on the [timeout manager](/nservicebus/messaging/timeout-manager.md) to implement [delayed delivery](/nservicebus/messaging/delayed-delivery.md). The transport now creates an infrastructure inside the broker to natively handle delaying messages.
+In Versions 4.3 and above, the RabbitMQ transport no longer relies on the [timeout manager](/nservicebus/messaging/timeout-manager.md) to implement [delayed delivery](/nservicebus/messaging/delayed-delivery.md). Instead, the transport creates infrastructure inside the broker which can delay messages using native RabbitMQ features.
```

Comment:
But that isn't directly relevant. It's just one piece of several different things that we've used to build this feature.